tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/queryexpressions/QueryExpressionTest.java

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
Shall we refactor the package name? `queryexpressions->expressions`

Reply:
Updated in 0aea9c28ba0c4281afbef57bc06d3ba1306f4d60